hub/tests/fixtures/geocoding_cases.py

```diff
@@ -0,0 +1,295 @@
+geocoding_cases = [
+    # Name matching; cases that historically didn't work
+    {
+        "id": "1",
+        "council": "Barnsley",
+        "ward": "St Helens",
+        "expected_area_type_code": "WD23",
+        "expected_area_gss": "E05000993",
+    },
+    {
+        "id": "2",
+        "council": "North Lincolnshire",
+        "ward": "Brigg & Wolds",
+        "expected_area_type_code": "WD23",
+        "expected_area_gss": "E05015081",
+    },
+    {
+        "id": "3",
+        "council": "Test Valley",
+        "ward": "Andover Downlands",
+        "expected_area_type_code": "WD23",
+        "expected_area_gss": "E05012085",
+    },
+    {
+        "id": "4",
+        "council": "North Warwickshire",
+        "ward": "Baddesley and Grendon",
+        "expected_area_type_code": "WD23",
+        "expected_area_gss": "E05007461",
+    },
+    # Name rewriting required
+    {
+        "id": "5",
+        "council": "Herefordshire, County of",
+        "ward": "Credenhill",
+        "expected_area_type_code": "WD23",
+        "expected_area_gss": "E05012957",
+    },
+    # GSS code matching
+    {
+        "id": "999",
+        "council": "E08000016",
+        "ward": "E05000993",
+        "expected_area_type_code": "WD23",
+        "expected_area_gss": "E05000993",
+    },
+    # Misc
+    # Gwynedd		Brithdir and Llanfachreth, Ganllwyd, Llanelltyd
+    # is Brithdir and Llanfachreth/Ganllwyd/Llanelltyd in MapIt
+    # https://mapit.mysociety.org/area/165898.html
+    {
+        "id": "6",
+        "council": "Gwynedd",
+        "ward": "Brithdir and Llanfachreth, Ganllwyd, Llanelltyd",
+        "expected_area_type_code": "WD23",  # TODO: actually it's a UTE, which
```

Comment:
suggestion: this comment looks incomplete

Reply:
Actually these comments are irrelevant. Removed.